### PR TITLE
Tamil translation added

### PR DESCRIPTION
**Pull request by manimaran96**

### CODE REVIEW

File: appintro/src/main/res/values-ta/strings.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="app_intro_skip_button">தவிர்</string>
+    <string name="app_intro_next_button" tools:ignore="MissingTranslation">அடுத்து</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-ta/strings.xml:4]:
```suggestion
    <string name="app_intro_next_button">அடுத்து</string>
```

File: appintro/src/main/res/values-ta/strings.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <string name="app_intro_back_button" tools:ignore="MissingTranslation">பின்னால்</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-ta/strings.xml:5]:
```suggestion
    <string name="app_intro_back_button">பின்னால்</string>
```

File: appintro/src/main/res/values-ta/strings.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    <string name="app_intro_done_button">முடிந்தது</string>
+    <string name="app_intro_image_content_description" tools:ignore="MissingTranslation">வரைகலை</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-ta/strings.xml:7]:
```suggestion
    <string name="app_intro_image_content_description">வரைகலை</string>
```

**Review by cortinico — CHANGES_REQUESTED** [review]
Thanks for the contribution 🙏  You just need to remove the `tools:ignore="MissingTranslation"` and we can merge it.

File: appintro/src/main/res/values-ta/strings.xml (hunk 1 above)
Comment (manimaran96) [reply appintro/src/main/res/values-ta/strings.xml]: Thanks @cortinico. 
Now updated.

**Review by cortinico — APPROVED** [review]

**Pull request merged**